src/components/NewbieSelect/NewbieSelect.tsx

```diff
@@ -17,17 +17,10 @@ const useStyles = makeStyles(theme => ({
   subTitle: {
     color: theme.palette.text.secondary,
   },
-  fabButton: {
-    position: 'fixed',
-    top: 'auto',
-    left: 'auto',
-    right: theme.spacing(3),
-    bottom: theme.spacing(3),
-  },
 }));
 
 const NewbieSelect: React.FC = () => {
-  const classes = useStyles();
+  const { subTitle } = useStyles();
```

Comment:
Since you touch this screen, you can optimize it. Please remove the useStyles and the makeStyles function. it's not needed anymore.